src/pages/components/typography.mdx

```diff
@@ -2,10 +2,159 @@
+
+Use Paragraph to render `<p>` elements with the appropriate Garden styling.
+
+import Paragraph from '../../examples/typography/Paragraph.tsx';
+import ParagraphCode from '!!raw-loader!../../examples/typography/Paragraph.tsx';
+
+<CodeExample code={ParagraphCode}>
+  <Paragraph />
+</CodeExample>
+
+### Type modifiers
+
+You can apply two modifiers to type: bold and monospace.
+
+import Modifiers from '../../examples/typography/Modifiers.tsx';
+import ModifiersCode from '!!raw-loader!../../examples/typography/Modifiers.tsx';
+
+<CodeExample code={ModifiersCode}>
+  <Modifiers />
+</CodeExample>
+
+### Span text
+
+Use Span to style and format `<span>` elements. You can apply bold and monospace to Span.
+
+import Span from '../../examples/typography/Span.tsx';
+import SpanCode from '!!raw-loader!../../examples/typography/Span.tsx';
+
+<CodeExample code={SpanCode}>
+  <Span />
+</CodeExample>
+
+### Code highlight
+
+Use Code to style and format `<code>` elements. You can change text size and background color. Code
+components come in small, medium, and large.
+
+import Code from '../../examples/typography/Code.tsx';
+import CodeCode from '!!raw-loader!../../examples/typography/Code.tsx';
+
+<CodeExample code={CodeCode}>
+  <Code />
+</CodeExample>
+
+### Lists
+
+Use Lists to style and format ordered (`<ol>`) and unordered (`<ul>`) lists. List components come
+in small, medium, and large.
+
+import Lists from '../../examples/typography/Lists.tsx';
+import ListsCode from '!!raw-loader!../../examples/typography/Lists.tsx';
+
+<CodeExample code={ListsCode}>
+  <Lists />
+</CodeExample>
+
+### Ellipsis text
+
+Use Ellipsis to automatically provide text content with a native title attribute and text-overflow
+styling.
+
+import Ellipsis from '../../examples/typography/Ellipsis.tsx';
+import EllipsisCode from '!!raw-loader!../../examples/typography/Ellipsis.tsx';
+
+<CodeExample code={EllipsisCode}>
+  <Ellipsis />
+</CodeExample>
+
+## Configuration
+
+<Configuration reactPackage={props.data.mdx.package} components={props.data.mdx.components} />
+
+## API
+
+### SM
+
+<PropSheet components={props.data.mdx.components} componentName="sm" />
+
+### MD
+
+<PropSheet components={props.data.mdx.components} componentName="md" />
+
+### LG
+
+<PropSheet components={props.data.mdx.components} componentName="lg" />
+
+### XL
+
+<PropSheet components={props.data.mdx.components} componentName="xl" />
+
+### XXL
+
+<PropSheet components={props.data.mdx.components} componentName="xxl" />
+
+### XXXL
+
+<PropSheet components={props.data.mdx.components} componentName="xxxl" />
+
+### Paragraph
+
+<PropSheet components={props.data.mdx.components} componentName="paragraph" />
+
+### Span
+
+<PropSheet components={props.data.mdx.components} componentName="span" />
+
+### Code
+
+<PropSheet components={props.data.mdx.components} componentName="code" />
+
+### OrderedList
+
+<PropSheet components={props.data.mdx.components} componentName="orderedlist" />
+
```

Comment:
Please add the H4 placeholder for `#### OrderedList.Item` as a prompt for filling out a description in the future. Compare with https://5f04a32ef33838130ff7274f--zendeskgarden.netlify.app/components/avatar#avatartext.